huaweicloud/services/cdn/resource_huaweicloud_cdn_domain.go

```diff
@@ -947,79 +949,77 @@ func getConfigsAttrs(hcCdnClient *cdnv1.CdnClient, domainName, epsId, privateKey
 		"ipv6_enable":                   configsResp.Ipv6Accelerate != nil && *configsResp.Ipv6Accelerate == 1,
 		"range_based_retrieval_enabled": analyseFunctionEnabledStatusPtr(configsResp.OriginRangeStatus),
 	}
-
-	sources = flattenSourcesAttrs(configsResp.Sources)
-	configs = []map[string]interface{}{configsAttrs}
-	cacheRules = flattenCacheRulesAttrs(configsResp.CacheRules)
-	return
+	return []map[string]interface{}{configsAttrs}
 }
 
-func getCacheAttrs(hcCdnClient *cdnv1.CdnClient, domainId, epsId string) ([]map[string]interface{}, error) {
-	req := model.ShowCacheRulesRequest{
-		DomainId:            domainId,
-		EnterpriseProjectId: utils.StringIgnoreEmpty(epsId),
-	}
-	resp, err := hcCdnClient.ShowCacheRules(&req)
+func queryDomainFullConfig(cfg *config.Config, d *schema.ResourceData, domainName string) (*model.ConfigsGetBody, error) {
+	hcCdnClient, err := cfg.HcCdnV1Client(cfg.GetRegion(d))
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error creating HC CDN v1 client: %s", err)
```

Comment:
the error message should not include `HC` or `HuaweiCloud`.

Reply:
OK